src/event/service/event.service.ts

```diff
@@ -484,16 +485,19 @@ class EventService implements BaseService {
         else return difference >= daysToEventInSeconds;
     };
 
-    private sortEventsByDistance = (event1: Partial<EventDetails>, event2: Partial<EventDetails>) => {
-        return event1.distance > event2.distance ? 1 : -1;
+    private sortEventsByDistance = (event1: Partial<EventDetails>, event2: Partial<EventDetails>, sort: SortDTO) => {
+        if (sort.isMoreOrLess == LOGICAL_OPERATION.MORE) return event1.distance < event2.distance ? 1 : -1;
+        else return event1.distance > event2.distance ? 1 : -1;
     };
 
-    private sortEventsByPopularity = (event1: Partial<EventDetails>, event2: Partial<EventDetails>) => {
-        return event1.participants < event2.participants ? 1 : -1;
+    private sortEventsByPopularity = (event1: Partial<EventDetails>, event2: Partial<EventDetails>, sort: SortDTO) => {
+        if (sort.isMoreOrLess == LOGICAL_OPERATION.LESS) return event1.participants > event2.participants ? 1 : -1;
+        else return event1.participants < event2.participants ? 1 : -1;
```

Comment:
if already returns so no need for else

Note: KPOP SUCKS BALLS

Reply:
report hate speech

Reply:
done